src/main/scala/higherkindness/rules_scala/workers/zinc/compile/AnalysisStore.scala

```diff
@@ -124,166 +141,340 @@ class AnxAnalyses(format: AnxAnalysisStore.Format) {
   }
 
   def sortAnnotation(annotation: Annotation): Annotation = {
-    annotation.copy(
-      arguments = annotation.arguments.sortWith(_.hashCode() > _.hashCode())
-    )
+    val sortedArguments = annotation.getArgumentsList.asScala.sortWith(_.hashCode() > _.hashCode()).asJava
+    Annotation
+      .newBuilder(annotation)
+      .clearArguments()
+      .addAllArguments(sortedArguments)
+      .build()
```

Comment:
The previous protobuf stuff was a Scala library. Their replacement is Java. So copy is not an option because it's not a Scala case class, rather it is a Java class.